apps/api/src/app/environments/usecases/update-environment/update-environment.command.ts

```diff
@@ -0,0 +1,21 @@
+import { IsDefined, IsMongoId, IsOptional, IsString, ValidateNested } from 'class-validator';
+import { OrganizationCommand } from '../../../shared/commands/organization.command';
+
+export class UpdateEnvironmentCommand extends OrganizationCommand {
+  @IsDefined()
+  @IsMongoId()
+  _id: string;
```

Comment:
Maybe we can call this `environmentId` instead of id? Might be easier to understand what it refers to.